runtime/near-vm-errors/src/lib.rs

```diff
@@ -1,17 +1,17 @@
 #![doc = include_str!("../README.md")]
 
-use std::fmt::{self, Error, Formatter};
-
 use borsh::{BorshDeserialize, BorshSerialize};
 use near_account_id::AccountId;
 use near_rpc_error_macro::RpcError;
 use serde::{Deserialize, Serialize};
+use std::any::Any;
+use std::fmt::{self, Error, Formatter};
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
The `PartialEq` and `Eq` impls here don't appear to be used at all. Is there any reason why we need to keep these implementations in place?

Comparing errors for equality (as opposed to matching their structure) in general seems like a pretty weird thing to need.

Reply:
If these are dropped, then the whole `AnyEq` thing could be replaced with just `Box<dyn std::error::Error + Send + Sync>` fields. Which is pretty common way to do this kind of type erasure in Rust.

Reply:
These are used in our tests :sob: And yeah, this is also problematic, as it, eg, prevents us from using `io::Error`. I do want to fix that (by rewriting our test), but not in this PR.